qh_inthash.c

```diff
@@ -297,12 +331,14 @@ static uint32_t qh_inthash_initialize_from_file(php_qh_inthash_obj *obj, php_str
 PHP_METHOD(QuickHashIntHash, loadFromFile)
 {
 	char *filename;
-	int   filename_len;
+	TYPE_ARG_L   filename_len;
 	long  size = 0, flags = 0;
 	php_stream *stream;
 
-	php_set_error_handling(EH_THROW, NULL TSRMLS_CC);
+	zend_error_handling error_handling;
```

Comment:
This line belongs to the declarations, so rather no newline before, but after instead.